src/carousel/carousel.component.html

```diff
@@ -1,13 +1,15 @@
-<div (mouseenter)="pause()" (mouseleave)="play()" (mouseup)="play()" class="carousel slide">
+<div (mouseenter)="pause()" (mouseleave)="play()" (mouseup)="play()" class="carousel slide" (focus)="pauseOnFocus()" tabindex="0">
```

Comment:
in this case carousel won't change sildes automatically on focus out. replace `focus` with `focusin` and `focusout`

Reply:
also it should be possible to disable this behavior via input like `[pauseOnFocus]`